core/spec/models/spree/calculator/refunds/default_refund_amount_spec.rb

```diff
@@ -3,30 +3,43 @@
 describe Spree::Calculator::Returns::DefaultRefundAmount, :type => :model do
   let(:order)           { create(:order) }
   let(:line_item_quantity) { 2 }
-  let(:pre_tax_amount)  { 100.0 }
-  let(:line_item)       { create(:line_item, price: 100.0, quantity: line_item_quantity, pre_tax_amount: pre_tax_amount) }
+  let(:price) { 100.0 }
+  let(:line_item)       { create(:line_item, price: price, quantity: line_item_quantity) }
   let(:inventory_unit) { build(:inventory_unit, order: order, line_item: line_item) }
   let(:return_item) { build(:return_item, inventory_unit: inventory_unit ) }
   let(:calculator) { Spree::Calculator::Returns::DefaultRefundAmount.new }
+  let(:tax_amount) { 0.2 }
 
-  before { order.line_items << line_item }
+  before do
+    create(:tax_rate, zone: create(:zone, default_tax: true),
+                      included_in_price: true,
+                      amount: tax_amount)
+    order.line_items << line_item
+    order.updater.update_totals
+  end
 
   subject { calculator.compute(return_item) }
 
   context "not an exchange" do
     context "no promotions or taxes" do
-      it { is_expected.to eq pre_tax_amount / line_item_quantity }
+      it { is_expected.to eq line_item.pre_tax_amount / line_item_quantity }
     end
 
     context "order adjustments" do
-      let(:adjustment_amount) { -10.0 }
+      let(:adjustment_amount) { order.item_total * -0.4 }
 
       before do
         order.adjustments << create(:adjustment, order: order, amount: adjustment_amount, eligible: true, label: 'Adjustment', source_type: 'Spree::Order')
         order.adjustments.first.update_attributes(amount: adjustment_amount)
+        order.updater.update_totals
       end
 
-      it { is_expected.to eq (pre_tax_amount - adjustment_amount.abs) / line_item_quantity }
+      it { is_expected.to eq (line_item.pre_tax_amount - adjustment_amount.abs) / line_item_quantity }
+
+      context 'refund amount * line item quantity with tax applied' do
```

Comment:
can we have two different specs, one for inclusive tax and one for additional tax